### PR TITLE
Add unplugin-icons and example of auto importing

### PR DESCRIPTION
**Pull request by samuveth**
Installing https://github.com/antfu/unplugin-icons#custom-icons

## How to use (for custom icons)

1. Add svg files to `assets/icons/*.svg`
2. Lazy load them from the icons folder image

3. Use it like a font and with auto import image

4. See the results 🐎 image

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -57,6 +57,7 @@
     "@typescript-eslint/parser": "^5.19.0",
     "@vitejs/plugin-vue": "^2.3.1",
     "@vitest/ui": "^0.9.3",
+    "@vue/compiler-sfc": "^3.2.33",
```
Comment (mktcode) [inline package.json:60]: I think this doesn't need to be installed separately. Comes with vue.

Comment (samuveth) [reply package.json]: Yep true :)

**Comment by mktcode** [comment]
This is not meant to be merged, right? More like a demo PR?

**Comment by bonustrack** [comment]
Nice! Can you try import the collection heroes icons outlined?

**Comment by mktcode** [comment]
Ok, so after discussing this with @samuveth I also come to the conclusion that this makes sense. It's way more flexible while at the same time making integration of new icons a lot easier, without the need for logging into a website to generate a new font file.

**Comment by samuveth** [comment]
> Nice! Can you try import the collection heroes icons outlined?

@bonustrack Yes, but aren't we using the `Phosphor` icon set in the design.

**Comment by bonustrack** [comment]
We only use Phosphor on the mobile app sketch I feel like hero icons are more cute, buth either way is fine with me just want to see how to setup collection to try on SX UI

**Review by mktcode — APPROVED** [review]
👍🏻

**Pull request merged**